hpo_pytorch_mnist/pytorch_mnist.ipynb

```diff
@@ -364,7 +364,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "predictor_hpo = tuner.deploy(initial_instance_count=1, instance_type='ml.m4.xlarge')"
```

Comment:
' train_instance_type='ml.m5.xlarge',\n",'

We have already prepared regular hands-on materials (pdf) with m4.xlarge of limit increase so that we don't want to change the instance type from m4.xlarge.